Makefile

```diff
@@ -140,6 +151,14 @@ $(TARGET).bin: $(TARGET).elf
 flash:
 	$(SF) write $(TARGET).bin 0x8000000
 
+ventilation_test: test/testRunner.c src/application/ventilation.o test/ventilation-test.o test/stubs/systemInfo_stub.o test/stubs/valve_stub.o test/stubs/dss_stub.o test/testUtil.o
+	$(CC) -o $@ $^
+	
+test: CC=$(CC_TEST)
```

Comment:
I personally find this a bit confusing... it seems that this re-assigns the variables in the Makefile for the test job?

Reply:
This is because I want to continue to use the code for the .o recipe